components/infra-proxy-service/server/servers.go

```diff
@@ -180,11 +180,17 @@ func (s *Server) GetServer(ctx context.Context, req *request.GetServer) (*respon
 	if err != nil {
 		return nil, service.ParseStorageError(err, *req, "server")
 	}
-	migration, err := s.service.Migration.GetLastSuccessfulMigration(ctx, req.Id)
+
+	lastMigration, err := s.service.Migration.GetLastSuccessfulMigration(ctx, req.Id)
```

Comment:
As we can't make one query for last successful migration and active-migrations, we have two queries